src/frontend/js/member.js

```diff
@@ -147,16 +147,40 @@ $(function() {
         }
     }
     function hideUiAccordionDays() {
-        $('.ui.accordion.meals').each(function () {
+        $('.ui.accordion.meals').not('.default').each(function () {
           $(this).hide();
         });
     }
 
     $("#form-meals-schedule select[multiple='multiple']").change(function () {
+
         hideUiAccordionDays();
         showUiAccordionSelectedDays();
     });
-    hideUiAccordionDays();
-    showUiAccordionSelectedDays();
+    var deliveryTypeSelect = $('#id_dietary_restriction-delivery_type, #id_delivery_type');
+    deliveryTypeSelect.change(function () {
+        if ($(this).val() == 'E') {
+            console.log('episodic');
+            $('#form-meals-schedule').hide();
+            hideUiAccordionDays();
+            $('.ui.accordion.meals.default').show();
+        }
+        else {
+            console.log('ongoing');
+            $('#form-meals-schedule').show();
+            showUiAccordionSelectedDays();
+            $('.ui.accordion.meals.default').hide();
+        }
+    });
+    if (deliveryTypeSelect.val() == 'E') {
```

Comment:
$(this) identify the object that has been changed in the callback function.
Though here, it's not in a callback function (as it's on page load), so I have to identify clearly my object.